<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGererCompteRecepteur</class>
 <widget class="QDialog" name="DialogGererCompteRecepteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>52</height>
      </size>
     </property>
     <property name="title">
      <string>Service </string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="rb_administration">
          <property name="text">
           <string>Administration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="comptabilite">
          <property name="text">
           <string>Comptabilité</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="reception">
          <property name="text">
           <string>Réception</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_restauration">
          <property name="text">
           <string>Restauration</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxNvRecept">
     <property name="title">
      <string>Enregistrer un compte</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Matricule:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMatricule"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nom:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditNom"/>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Résidence:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditResidence"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Prénom(s):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPrenoms"/>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>11</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Contacts:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditContacts">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="btn_contacts">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sexe précis:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Monsieur">
          <property name="text">
           <string>M.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Madame">
          <property name="text">
           <string>Mme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Demoiselle">
          <property name="text">
           <string>Mlle</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <layout class="QFormLayout" name="formLayout">
        <property name="formAlignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_enregistrer">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Enregistrer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_initialiser">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Initialiser</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Comptes du service sélectionné</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Comptes</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <column>
             <property name="text">
              <string>Statut</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Matricule</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Sexe précis</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nom</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Prénoms</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Cel</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tel</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>E-mail</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Boîte postale</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Résidence</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QPushButton" name="btn_initialiserCompte">
             <property name="maximumSize">
              <size>
               <width>79</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Initialiser</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_supprimer">
             <property name="maximumSize">
              <size>
               <width>79</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_editer">
             <property name="maximumSize">
              <size>
               <width>79</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Editer</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_suspendre_activer">
             <property name="maximumSize">
              <size>
               <width>97</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Suspendre/Activer</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_actualiser">
             <property name="maximumSize">
              <size>
               <width>79</width>
               <height>23</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Actualiser</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_fermer">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Fermer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
